src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -200,15 +200,32 @@ export default {
 <style lang="scss" scoped>
 @import 'settings';
 
+$row-max-width: 63.75rem;
+
 .cards-and-arrows-wrapper {
+	max-width: $row-max-width;
+	margin: 0 auto;
 	align-items: center;
 	display: flex;
+	position: relative;
+	justify-content: center;
 }
 
 .arrow {
```

Comment:
New html markup and design for arrows which are now semi transparent. The arrows also dictate the offsets for page margins as they had a hard requirement to fall "outside" of the cards on the left. This requirement sets the left position of the first loan card which dictates the title position of each row and requires special margins on the header region.

*Arrow and row styles continue to be duplicated in feature rows for the moment which is current state. I do see a path however for consolidation of the layouts and styles.